src/pulse3D/plate_recording.py

```diff
@@ -374,6 +375,28 @@ def _process_plate_data(self, calibration_recordings, use_mean_of_baseline):
             well_file.displacement = np.array([adjusted_time_indices, x])
             well_file.force = calculate_force_from_displacement(well_file.displacement)
 
+    def _write_time_force_csv(self, output_dir: str):
```

Comment:
Probably want to remove the leading underscore from this method name to indicate that it's public since it will primarily be used by users of this class

Reply:
yes, my bad